src/index.tsx

```diff
@@ -350,6 +347,6 @@ export const TopologyPipelinesGettingStartedDemo: React.FC = () => {
         </VisualizationProvider>
     );
 };
-
-const container = document.getElementById("root");
-createRoot(container).render(<TopologyPipelinesGettingStartedDemo />);
+//
+// const container = document.getElementById("root");
+// createRoot(container).render(<TopologyPipelinesGettingStartedDemo />);
```

Comment:
I removed this because RHDH creates its own root.  It essentially embeds a bunch of stuff on one page.  If we want to run this library standalone, then we would need to recreate this section elsewhere and leave it out of the `main` section in `package.json` so RHDH doesn't pick it up

Reply:
That's fine. I can uncomment when working locally, or some day can figure out how to run this with Backstage